### ISSUE TITLE
4.03 compatibility

### ISSUE THREAD

**Issue by copy** [mirage/ocaml-cstruct#97]
Trying to build on 4.03.0+flambda using opam gives this error:

```
File "ppx/ppx_cstruct.ml", line 301, characters 30-40:
Error: This variant expression is expected to have type Parsetree.constant
       The constructor Const_char does not belong to type Parsetree.constant
Hint: Did you mean Pconst_char?
Command exited with code 2.
```

**Comment by hannesm** [comment]
see #96

**Comment by marklrh** [comment]
addressed in #96

**Comment by copy** [comment]
Whoops, missed that.
